examples/pytorch-keyworddetection-api/README.rst

```diff
@@ -0,0 +1,62 @@
+FEDn Project: Keyword Detection (PyTorch)
+-----------------------------
+
+This is an example to showcase how to set up FEDnClient and use APIClient to setup and manage a training from python. 
+The machine learning project is based on the Speech Commands dataset from Google. 
```

Comment:
maybe add link to dataset explaination: https://huggingface.co/datasets/google/speech_commands